test/parallel/test-buffer-write-noassert.js

```diff
@@ -11,12 +11,21 @@ function write(funx, args, result, res) {
     assert.deepStrictEqual(buf, res);
   }
 
+  /*{
+    const invalidArgs = Array.from(args);
+    invalidArgs[1] = -1;
+    assert.throws(
+      () => Buffer.alloc(9)[funx](...invalidArgs),
+      /^RangeError: Index out of range$/
+    );
+  }*/
+
   {
     const invalidArgs = Array.from(args);
     invalidArgs[1] = -1;
     assert.throws(
       () => Buffer.alloc(9)[funx](...invalidArgs),
-      /^RangeError: (?:Index )?out of range(?: index)?$/
+      /^RangeError(?:\[ERR_INDEX_OUT_OF_RANGE\])?: Index out of range$/
```

Comment:
+1... we should probably look at fixing up `common.expectsError()` to account for this, however. Will have to think on that for a bit